alembic/versions/735063d71b57_added_document_uploads_and_team_deletion.py

```diff
@@ -0,0 +1,56 @@
+"""Added document uploads and team deletion
+
+Revision ID: 735063d71b57
+Revises: c31b1a94a27d
+Create Date: 2017-11-26 00:16:52.930412
+"""
+
+# revision identifiers, used by Alembic.
+revision = '735063d71b57'
+down_revision = 'c31b1a94a27d'
```

Comment:
I confirmed that this is the correct value, so we're not affected by that migration branching bug.